File: VB/dxSample/Custom Tree List/CustomTreeList.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraTreeList

Namespace dxSample
    Friend Class CustomTreeList
        Inherits TreeList

        Protected Overrides Function CreateViewInfo() As DevExpress.XtraTreeList.ViewInfo.TreeListViewInfo
            Return New CustomTreeListViewInfo(Me)
        End Function
    End Class


End Namespace

File: CS/dxSample/Custom Tree List/CustomTreeList.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraTreeList;

namespace dxSample {
    class CustomTreeList : TreeList {
        protected override DevExpress.XtraTreeList.ViewInfo.TreeListViewInfo CreateViewInfo() {
            return new CustomTreeListViewInfo(this);
        }
    }
    

}
